applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,264 @@
+.. _nrf_peripheral_sensor_node_app:
+
+nRF Peripheral Sensor node
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Peripheral Sensor node is an application that periodically acquires data from a Serial Peripheral Interface (SPI) and Two-wire Interface (TWI) devices, advertises, and enables the reading of sensor values from Bluetooth® service in a power-efficient way.
```

Comment:
```suggestion
The nRF Peripheral Sensor node is an application that periodically acquires data from a Serial Peripheral Interface (SPI) and Two-wire Interface (TWI) devices. 
It advertises and enables the reading of sensor values from a Bluetooth® service in a power-efficient way
```

Reply:
> 

```suggestion
The nRF Peripheral Sensor node is an application that periodically acquires data from devices using either the Serial Peripheral Interface (SPI) or Two-wire Interface (TWI), advertises, and enables the reading of sensor values from Bluetooth® services in a power-efficient way.
```
It works on both SPI and TWI devices, right? @rakons 
What's the bluetooth service it enables the reading of sensor values from?
Is it a specific one?